tools/edta/edta.xml

```diff
@@ -0,0 +1,472 @@
+<tool id="edta" name="edta" version="@WRAPPER_VERSION@+@VERSION_SUFFIX@" profile="20.01">
+    <description>
+        Whole-genome de-novo TE annotation 
+    </description>
+    <macros>
+        <token name="@WRAPPER_VERSION@">2.0.0</token>
+        <token name="@VERSION_SUFFIX@">galaxy0</token>
+        <import>edta_macros.xml</import>
+    </macros>
+    <expand macro="bio_tools"></expand>
+    <requirements>
+        <container type="docker">oushujun/edta:@WRAPPER_VERSION@</container>
```

Comment:
@tcollins2011 the conda package can not be used? Sorry, I do not remember why this was a problem :(